cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -42,706 +40,221 @@
+
+    required.add_argument('--job_number',
+                          dest="job_number",
+                          action='store',
+                          help='The Jenkins build number',
+                          required=True)
+
+    optional.add_argument('--config',
+                          dest='configfile',
+                          action='store',
+                          default=os.path.join(cwd, "pr_config/pullrequest.ini"),
+                          help="Configuration file. Default=%(default)s",
+                          required=False)
+
+    optional.add_argument('--workspaceDir',
+                          dest="workspaceDir",
+                          action='store',
+                          default=default_workspace,
+                          help="The local workspace directory that Jenkins set up." +
+                               " Default={}".format(default_workspace),
+                          required=False)
+
+    optional.add_argument('--packageEnables',
+                          dest="package_enables",
+                          action="store",
+                          default=default_package_enables,
+                          help="Custom packageEnables.cmake override." +
+                               " Default={}".format(default_package_enables))
+
+    optional.add_argument('--subprojects_file',
+                          dest="subprojects_file",
+                          action="store",
+                          default=default_subprojects_file,
+                          help="Custom subprojects file." +
+                               " Default={}".format(default_subprojects_file))
+
+    optional.add_argument('--mode',
+                          dest='mode',
+                          action='store',
+                          default='standard',
+                          help="PR testing mode. 'standard' for normal PR tests, 'installation'" +
+                               " for installation testing. Default = %(default)s")
+
+    optional.add_argument('--req-mem-per-core',
+                          dest='req_mem_per_core',
+                          action='store',
+                          default=3.0,
+                          help="Minimum required memory per core (Gigabytes)." +
+                               "Default = %(default)s")
+
+    optional.add_argument('--max-cores-allowed',
+                          dest='max_cores_allowed',
+                          action='store',
+                          default=12,
+                          help="Max cores allowed, if >= 0 we will use the # of " +
+                               "detected cores on the system. Default = %(default)s")
+
+    optional.add_argument('--num-concurrent-tests',
+                          dest='num_concurrent_tests',
+                          action='store',
+                          default=-1,
+                          help="Number of concurrent tests to run. Default = %(default)s")
```

Comment:
What does -1 mean as the default.

Reply:
I updated the documentation.

This is equivalent to `ctest -j <num_concurrent_tests>` ... if the value is >= 0 then it's considered an override and that value is used. Otherwise, it will compute the parameter via `num_cores / max_test_parallelism`